mailqueue/tasks.py

```diff
@@ -8,7 +8,18 @@
 @shared_task(name="tasks.send_mail", default_retry_delay=5, max_retries=5)
 def send_mail(pk):
     message = MailerMessage.objects.get(pk=pk)
-    message._send()
+    try:
+        message._send()
+    except Exception as e:
+        # Assuming that Django project uses anymail.backends.postmark.EmailBackend
+        if e.__class__.__name__ == "AnymailRecipientsRefused":
+            # Postmark API had refused recipient's email address.
+            # Do not retry the task, in this case there's no need to hit API repeatedly.
+            # Do not report this to Sentry. Refused recipients are tracked in Postmark UI as suppressed.
+            return
+        else:
+            # On any other exception proceed further to retry.
+            pass
```

Comment:
I feel this is redundant. 
```suggestion
```

Reply:
Say this is not an error related to attachment, but to anything what may fail in the `_send` method.
If we'll just log it and not report the exception - then we will not be aware about it, as we don't review logs as often as we review sentry exceptions.
Why not to report these `MaxRetriesExceededError` to sentry?
It does not happen too often.

Reply:
We may also setup datadog alerts for this exception in logs, but then we have to do that and also it would have similar purpose to Sentry notification. I think that after all, we should be aware that this exception happened, otherwise we'll be notified by users or support team, instead of Sentry.

Reply:
> Say this is not an error related to attachment, but to anything what may fail in the `_send` method.
This is too hypothetical for me. 

Till now 100% of the errors were related to 
`Mail Queue Exception: All message recipients were rejected or invalid`

Reply:
If you are worried about other failures, the better solution is to add an alert in Datadog (thats what it is for), instead of creating more technical debt in code.